src/config/defaults.js

```diff
@@ -9,10 +9,8 @@ export default {
   playsinline: videojs.browser.IS_IOS,
   skin: 'dark',
   controls: false,
-  controlBar: {
-    pictureInPictureToggle: false,
-    chaptersButton: false
-  },
+  chaptersButton: false,
+  pictureInPictureToggle: false,
```

Comment:
these are no longer from videojs perspective part of the controlBar?

Reply:
they are the same buttons, children of the controlBar, only that we expose the setting as part of our root config